### PR TITLE
feature/MTM-44234/MTM-45350-Update dcumentation

### PR DESCRIPTION
**Pull request by Airdroplet**

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-permissions.md [hunk 1]
```diff
@@ -309,9 +309,9 @@ The following default inventory roles are initially available in new tenants:
 
 #### To add an inventory role
 
-Click **Add Role** in the **Inventory roles** tab.
+Click **Add inventory role** in the **Inventory roles** tab. 
 
-At the top of the page you can edit the name of the inventory role. Click on the name, edit it and click the green checkmark to save your edits.
+Form will be displayed, asking for **name**, **description** and **permissions** assignment for new inventory role.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-permissions.md:314]: Form will be displayed, asking for **name**, **description** and **permissions** assignment for new inventory role.
Better: In the **New inventory role** page, provide a name and a description, and assign the permissions for the new inventory role.

File: content/users-guide/administration-bundle/managing-permissions.md [hunk 2]
```diff
@@ -375,7 +375,7 @@ You can also copy inventory roles from another user. To copy roles, click **Copy
 
 If you try to perform actions without sufficient permissions, an error message will occur.
 
-To help troubleshooting permissions, click the **User** button at the right of the top bar. From the context menu, select **Access denied requests**. In the resulting window details on the denied accesses are provided. An administrator user or the [product support](/welcome/contacting-support/) can help in fixing the permissions.
+To help troubleshooting permissions, click on the current **User** name at the right of the top bar. From the context menu, select **Access denied requests**. In the resulting window details on the denied accesses are provided. An administrator user or the [product support](/welcome/contacting-support/) can help in fixing the permissions.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-permissions.md:378]: Not sure, if this change makes sense :-). It is actually both, the user button and/or the current user name. But bold formatting is wrong, as the button does not have "User" as label. So perhaps: 
Click on the User button (showing the current user name) at the right of the top bar

Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-permissions.md:378]: Forget what I said. Just looked it up in the style guide. Buttons without label are also formatted bold. So:
Click the **User** button (showing the current user name) at the right of the top bar

Comment (Airdroplet) [reply content/users-guide/administration-bundle/managing-permissions.md:378]: applied proposal

File: content/users-guide/administration-bundle/managing-permissions.md (hunk 1 above)
Comment (Airdroplet) [reply content/users-guide/administration-bundle/managing-permissions.md:314]: proposal applied

File: content/users-guide/administration-bundle/managing-permissions.md (hunk 2 above)
Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-permissions.md]: True, this is not a normal button. If it had the label "User" we would just say "Click **User**" without mentioning the button at all which is not possible here. I think the solution is good as is now.

**Review by BeateRixen — APPROVED** [review]

**Review by jdreSAG — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Airdroplet Should I merge this PR and add to 10.14 branch?

**Comment by Airdroplet** [comment]
> 

@BeateRixen yes, you can do so. All changes were intended for 10.14

**Comment by Airdroplet** [comment]
@BeateRixen There is an issue with one screenshot being noticed by @pawel-rynarzewski-sag , will push update in a moment.

**Comment by Airdroplet** [comment]
> @BeateRixen There is an issue with one screenshot being noticed by @pawel-rynarzewski-sag , will push update in a moment.

One more fix here: https://github.com/SoftwareAG/c8y-docs/pull/411/files

**Pull request merged**